packages/front/src/lib/components/MultiStepForm/MultiStepForm.svelte

```diff
@@ -5,9 +5,9 @@
 
     export let steps = [];
     export let onSubmit;
-    export let submitLabel = "Submit";
-    export let nextLabel = "Next";
-    export let previousLabel = "Previous";
+    export let submitLabel = "Soumettre";
```

Comment:
```suggestion
    export let submitLabel = "Confirmer";
```
:innocent: 

En français ça me fait bizarre "soumettre"